pkgs/development/haskell-modules/configuration-ghcjs.nix

```diff
@@ -79,12 +79,6 @@ self: super: {
   ghcjs-dom = overrideCabal super.ghcjs-dom (drv: {
     buildDepends = [ self.base self.mtl self.text self.ghcjs-base ];
```

Comment:
> (still, some overrides in the dependencies are needed to allow it to build both in ghc and ghcjs)

I assume you mean this? I trust your judgement, but out of curiosity why is that so?